changlog.md

```diff
@@ -0,0 +1,33 @@
+# unreleased
+
+* Automatically configure analytics. Projects no longer need a copy of
```

Comment:
Back-filled older release notes, but going forward, PRs should update this file whenever we make meaningful adjustments so we don't forget what we've changed.